packages/social-connection/schemas/SocialConnection.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "title": "SocialConnection",
+  "type": "object",
+  "$defs": {
+    "connection": {
+      "type": "object",
+      "properties": {
```

Comment:
Can any of these properties be required please? That would make the schema easier to use.

Reply:
for sure! changed

Reply:
Thanks!